inc/saiport.h

```diff
@@ -221,6 +221,19 @@ typedef enum _sai_port_fec_mode_extended_t
     SAI_PORT_FEC_MODE_EXTENDED_FC,
 } sai_port_fec_mode_extended_t;
 
+/**
+ * @brief Attribute data for #SAI_PORT_ATTR_DATAPATH_ENABLE
+ */
+typedef enum _sai_port_datapath_enable_t
```

Comment:
```suggestion
typedef enum _sai_port_pcs_datapath_mode_t
```

Reply:
In future we can add other options that's why it is not limited to only PCS mode.